packages/docker-engine/docker-engine.spec

```diff
@@ -3,7 +3,7 @@
 %global goorg github.com/docker
 %global goimport %{goorg}/docker
 
-%global gover 20.10.7
+%global gover 20.10.11
 %global rpmver %{gover}
 %global gitrev b0f5bc36fea9dfb9672e1e9b1278ebab797b9ee0
```

Comment:
I think we need to update this git commit to match the release version - 847da184ad5048b27f5bdf9d53d070f731b43180

Reply:
Nice catch!